src/proc/color-formats-converter.cpp

```diff
@@ -18,7 +18,7 @@
 #include <tmmintrin.h> // For SSSE3 intrinsics
 #endif
 
-#if defined (ANDROID) || (defined (__linux__) && !defined (__x86_64__))
+#if defined (ANDROID) || (defined (__linux__) && !defined (__x86_64__)) || ((defined(__arm64__) && defined(__APPLE__)) || defined(__aarch64__))
```

Comment:
Thanks for the PR,
Can you explain why __aarch64__ was added?
As I understand apple M1 will report __arm64__ right?

Reply:
I'll be honest, I can't remember - happy for you to remove that bit of the condition if it still works 😂

Reply:
Ah, looks like it's already present now anyways - closing this MR :)

Reply:
Thanks for the hard work.
We still having issues getting libusb to work on MAC, hope this we resolve..